.github/workflows/ci.yml

```
@@ -102,20 +102,20 @@ jobs:
     name: Typos
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: stable
           override: true
       - name: Typos check with custom config file
-        uses: crate-ci/typos@master
+        uses: crate-ci/typos@v1
```

I assumed `v1` would work given there is a [recent `1.2` release](https://github.com/crate-ci/typos), but that is not the case.

Reverting:

```suggestion
        uses: crate-ci/typos@master
```